### DISCUSSION TITLE
我之前通过一个空洞去了另一边

### DISCUSSION THREAD

**Discussion by jjxya** [share121/inter-knot#1299] [其他]
![cad6e5cb246e20b72622ac4f2d41ef4](https://github.com/user-attachments/assets/68508c94-2b35-4b02-921d-434897680f00)
各位绳匠们，我前段时间通过一个空洞去了另一个地方，过了一段时间我才知道空洞不知道把我传到哪个世界了，过了几年（谁家空洞把人困了几年啊！）
结果最后整个世界都没多少人了！话说有谁知道这个地方是哪里吗，他们说着崩坏啊，律者啊什么的根本听不懂啊，但总感觉有些熟悉。
哦对，不得不说那个地方好危险，还好他们说我有什么崩坏能适应性（为什么我不是这个世界的人会有崩坏能适应性啊？！），运气不错成了什么融合战士？
最后准备迎接决战时我们互相送了礼物，一位叫爱莉的美少女送给我一朵水晶花，那个看起来就非常强的凯文老大送了一个仿制天火圣裁，听他说是维尔薇她们做的。本来科斯魔准备送我一个口琴的，结果我还没接到就突然出现一个空洞把我传会来了。
话说你们见过这样的空洞吗？没听说过还有这种性质的空洞啊？希望还能再次遇见这样的空洞。告别的话我还没说呢！梅博士给我的火种计划怎么办啊！
啊啊啊，我好想再过去啊！

**Comment by Xieleng42** [comment]
应该是“逐火之蛾”，不用担心。如果实在害怕可以去找一个叫符华的人。

**Comment by Xieleng42** [comment]
但你可能再也不会到那了。

**Comment by Inter-Knot-User** [comment]
这话说出来你可能伤心了，我一个黄头发的朋友说只有一个神舟人活了下来

**Reply by xianyujun4** [reply]
别担心，你会亲眼见证”他们的故事“
